drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -520,6 +520,18 @@ static int nrf5_tx(const struct device *dev,
 
 	LOG_DBG("Result: %d", nrf5_data.tx_result);
 
+#if NRF_802154_ENCRYPTION_ENABLED
+	/*
+	 * When frame encryption by the radio driver is enabled, the frame stored in the tx_psdu
+	 * buffer is authenticated and encrypted in place. After an unsuccessful TX attempt,
```

Comment:
This was just added couple of days ago, why there is need to change it now? The comment says that this is done after an unsuccessful TX attempt, so why is it necessary to do it also after successful transmit? If this is ok, then the comment needs to be changed.

Reply:
I think that the comment is ok, I extended it with new, not seen before, case. Look at the last sentence of the comment.